google/cloud/firestore_v1/async_client.py

```diff
@@ -86,6 +92,28 @@ def __init__(
             client_options=client_options,
         )
 
+    @property
+    def _firestore_api(self):
+        """Lazy-loading getter GAPIC Firestore API.
+        Returns:
+            :class:`~google.cloud.gapic.firestore.v1`.async_firestore_client.FirestoreAsyncClient:
+            The GAPIC client with the credentials of the current client.
+        """
+        return self._firestore_api_helper(
+            firestore_grpc_transport.FirestoreGrpcAsyncIOTransport,
+            firestore_client.FirestoreAsyncClient,
+            firestore_client,
+        )
+
+    @property
+    def _target(self):
+        """Return the target (where the API is).
```

Comment:
What does target/location mean in this context? might be helpful to give an example.

Reply:
Added example!